### DISCUSSION TITLE
Call ReadAlone programmatically

### DISCUSSION THREAD

**Discussion by bluesidra** [ken107/read-aloud#367] [General]
Hello!

Is there a function to hand ReadAloud a string to read by another extension? Or even call the standard-function/popup programmatically?
I'm tinkering on an extension that selects text that I want to have read out loud, and it would save me a bunch of clicks if I could trigger ReadAloud automatically.

thanks,
blue

**Comment by ken107** [comment]
We don't have an external API at this time. PR is welcome, just add handlers for `chrome.runtime.onMessageExternal.addListener` in "events.js". Thanks
